kong/tools/faker.lua

```diff
@@ -33,16 +33,27 @@ Faker.FIXTURES = {
     { name = "API TESTS 7", public_dns = "cors2.com", target_url = "http://mockbin.com" },
     { name = "API TESTS 8 (logging)", public_dns = "logging.com", target_url = "http://mockbin.com" },
 
-    { name = "API TESTS 8 (dns)", public_dns = "dns1.com", target_url = "http://127.0.0.1:7771" },
-    { name = "API TESTS 9 (dns)", public_dns = "dns2.com", target_url = "http://localhost:7771" },
+    { name = "API TESTS 9 (dns)", public_dns = "dns1.com", target_url = "http://127.0.0.1:7771" },
+    { name = "API TESTS 10 (dns)", public_dns = "dns2.com", target_url = "http://localhost:7771" },
+    { name = "API TESTS 11 (multiple rate limiting)", public_dns = "test8.com", target_url = "http://mockbin.com" },
 
     -- DEVELOPMENT APIs. Please do not use those in tests
     { name = "API DEV 1", public_dns = "dev.com", target_url = "http://mockbin.com" },
   },
   consumer = {
     { custom_id = "provider_123" },
-    { custom_id = "provider_124" }
+    { custom_id = "provider_124" },
+    { custom_id = "provider_125" },
   },
+  keyauth_credential = {
```

Comment:
Why is this here? They are supposed to be under `plugins_configuration` and the comment "TODO: remove plugins from core" is **not** about `plugins_configuration`, but about the `keyauth_credentials` and `basicauth_credentials`.